public/js/views/projectView.js

```diff
@@ -3,28 +3,50 @@
 var projectView = {};
 
 projectView.handleMainNav = function () {
-  $('.main-nav').on('click', '.tab', function () {
-    // from starter code lab 5:
-    // if ($(this) === $('#main')) {
-    //   $('#list-stuff').hide();
-    //   $('#about').hide();
-    //   $('#projects').hide();
-    // } else if ($(this) === $('#about')) {
-    //   $('#main').hide();
-    //   $('#list-stuff').hide();
-    //   $('#projects').hide();
-    // } else {
-    //   $('#main').hide();
-    //   $('#about').hide();
-    //   $('#list-stuff').hide();
-    // }
-    // $('.tab-content').hide();
-    // from starter code lab 4:
-    // $('#' + $(this).data('content')).fadeIn();
+  // The following content is help from Aeone:
+  // =================click on nav-item hides other content
```

Comment:
You can delete the comments I put on there. That way you don't get dinged for it.